seatunnel-connectors-v2/connector-clickhouse/src/main/java/org/apache/seatunnel/connectors/seatunnel/clickhouse/sink/file/ClickhouseFileSinkFactory.java

```diff
@@ -44,6 +67,77 @@ public String factoryIdentifier() {
         return "ClickhouseFile";
     }
 
+    @Override
+    public TableSink<SeaTunnelRow, ClickhouseSinkState, CKFileCommitInfo, CKFileAggCommitInfo>
+            createSink(TableSinkFactoryContext context) {
```

Comment:
I think we should add a test case to cover this part. To make sure the sink can be created normally. We can do it in Clickhouse E2E (there has clickhouse container). CC @liunaijie @Carl-Zhou-CN

Reply:
support